src/main/java/spoon/compiler/builder/ClasspathOptions.java

```diff
@@ -42,22 +44,26 @@ public T classpath(String... classpaths) {
 	}
 
 	public T classpathFromListOrClassLoader(String... classpaths) {
-		return (classpaths != null && classpaths.length > 0) ? classpath(classpaths) : classpathFromCurrentClassLoader();
+		return classpath(getClasspathFromListOrCurrentClassloader(classpaths));
 	}
 
-	public T classpathFromCurrentClassLoader() {
+	public static String[] getClasspathFromListOrCurrentClassloader(String... classpaths) {
+		return (classpaths != null && classpaths.length > 0) ? classpaths : getClasspathFromCurrentClassloader();
+	}
+	public static String[] getClasspathFromCurrentClassloader() {
```

Comment:
Good point! It makes the usage of ClasspathOptions more complicated. I will move these static methods to place where I need them, so they do not make ClasspathOptions more compex.